xapian-core/generate-exceptions

```diff
@@ -21,7 +21,9 @@ use strict;
 use exception_data qw(
     $copyright $generated_warning @baseclasses @classes %classcode
 );
+use File::Path qw(make_path);
 
+make_path("include/xapian");
```

Comment:
The appveyor CI build for mingw fails with:

```
/usr/bin/perl -w -I"." "./generate-exceptions"
"make_path" is not exported by the File::Path module
Can't continue after import errors at ./generate-exceptions line 24
```

It looks like `make_path` is the new name and this used to be called `mkpath` before File::Path 2.00.

According to `corelist -a File::Path`, perl 5.9.4 with File::Path 1.08 is the most recent version without `make_path`.  That's a pretty ancient perl version but I think just changing `make_path` to `mkpath` would allow this to work there (for a single directory argument they should do the same thing) so I guess we should if only to keep the CI working.

Reply:
Actually, I think I misread the docs and I think it's before File::Path 2.08 which means before Perl 5.11.1.